samples/client/petstore/typescript-redux-query/builds/default/README.md

```diff
@@ -3,7 +3,7 @@
 This generator creates TypeScript/JavaScript client that utilizes [redux-query](https://amplitude.github.io/redux-query/).
 The generated Node module does not depend on ReactJS specifically.
 
-It can be used in both TypeScript and JavaScript. In TypeScript, the definition should be automatically resolved via `package.json`. ([Reference](http://www.typescriptlang.org/docs/handbook/typings-for-npm-packages.html))
+It can be used in both TypeScript and JavaScript. In TypeScript, the definition should be automatically resolved via `package.json`. ([Reference](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html))
```

Comment:
looks like it's already done in the master

```
## {{npmName}}@{{npmVersion}}

This generator creates TypeScript/JavaScript client that utilizes [redux-query](https://amplitude.github.io/redux-query/).
The generated Node module does not depend on ReactJS specifically.

It can be used in both TypeScript and JavaScript. In TypeScript, the definition will be automatically resolved via `package.json`. ([Reference](https://www.typescriptlang.org/docs/handbook/declaration-files/consumption.html))

```

so closing this one

Reply:
Ah. Missed that. Thank you.